config/v1/tests/nodes.config.openshift.io/AAA_ungated.yaml

```diff
@@ -12,3 +12,99 @@ tests:
         apiVersion: config.openshift.io/v1
         kind: Node
         spec: {}
+    - name: Should be able to create a Node object with cgroupMode set to "v2"
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "v2"
+      expected: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "v2"
+    - name: Should not allow to create a Node object with the cgroupMode set to "v1"
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "v1"
+      expectedError: "cgroupMode \"v1\" is not supported, please update to cgroupMode \"v2\""
+    - name: Should not allow to create a Node object with the cgroupMode set to some random value
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "unknown"
+      expectedError: "spec.cgroupMode: Unsupported value: \"unknown\""
+  onUpdate:
+    - name: Should be able to update a Node object with cgroupMode set to "v2"
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec: {}
+      updated: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "v2"
+      expected: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "v2"
+    - name: Should not allow update of cgroupMode from "v2" to "v1"
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "v2"
+      updated: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "v1"
+      expectedError: "cgroupMode \"v1\" is not supported, please update to cgroupMode \"v2\""
+    - name: Should allow changing other fields when a persisted value is no longer valid
```

Comment:
Please make sure this includes the cgroups keyword so we can distinguish it from other cases